### ISSUE TITLE
Squiggles for invalid settings

### ISSUE THREAD

**Issue by alexdima** [microsoft/vscode#176665]
When using something like the following to the settings:
```
"something": { "*": 'DEBUG' }
```
This is a typo for:
```
"something": { "*": "DEBUG" }
```
The setting is not really available, but there is no squiggle shown around `'DEBUG'`

**Comment by alexdima** [comment]
You're right, in my Insiders I see no problems in my settings.json even with syntax errors. But I do see squiggles in Stable.

**Comment by alexdima** [comment]
This reproduces when using `"workbench.settings.openDefaultSettings": true`

**Comment by aeschli** [comment]
To reproduce
- set "workbench.settings.openDefaultSettings": true,
- close the settings editor
- run `Preferences: Open User Settings (JSON)`. Opens a split editor
- add some typos. No syntax errors

@dbaeumer The JSON LS waits for diagnostics to be pulled. My guess is that the code that tracks the open documents doesn't know how to get the document URI's of the split editor. The editor is registered here:
https://github.com/microsoft/vscode/blob/ea7fca78270b90cad6952795dce03e0336612cb3/src/vs/workbench/contrib/preferences/common/preferencesContribution.ts#L62

**Comment by dbaeumer** [comment]
I access the document using the `Tab` API. I debugged it and the reason for the bug is that the input of the tab representing the split editor is `undefined`. Hence I can't determine its Uri

<img width="960" alt="image" src="https://user-images.githubusercontent.com/1931590/231074343-7aacc508-b6ff-408d-8ecf-9964ff0fbe7e.png">

@loganrosen can you please let me know how this should work for split editors?

**Comment by lramos15** [comment]
I'll take a look, thanks for the pointer
